tmt/convert.py

```diff
@@ -81,6 +81,17 @@ def read(path, makefile, nitrate, purpose):
         data['duration'] = re.search(
             r'echo "TestTime:\s*(.*)"', content).group(1)
         echo(style('duration: ', fg='green') + data['duration'])
+        # Requires and RhtsRequires (optional)
+        try:
+            data['requires'] = re.search(
+                r'echo "Requires:\s*(.*)"', content).group(1)
+            data['requires'] += ' ' + re.search(
+                r'echo "RhtsRequires:\s*(.*)"', content).group(1)
+        except AttributeError:
+            pass
+        if data['requires']:
+            echo(style('requires: ', fg='green') + data['requires'])
```

Comment:
This results in traceback if there is no `Requires` in the Makefile:

    File "/home/psss/git/tmt/tmt/cli.py", line 418, in convert
        data = tmt.convert.read(path, makefile, nitrate, purpose)
    File "/home/psss/git/tmt/tmt/convert.py", line 92, in read
        if data['requires']:
    KeyError: 'requires'

This might happen if test is using only hard `RhtsRequires`.  Please handle this case as well.

Reply:
changed in a0e0001